TwinePlatform/twine-monolith
# 📱 Native app - Volunteer side styles

relates #338 

### Changes
- add script for creating component files
- add `native-base` as componenent library
	- `react-native-elements` did not have a picker/dropdown option and was over complicated to make vanilla
- replace all `react-native-elements` componenets
- add Time page
- add Settings page
- setup test suite for CI but no tests yet. Will add when this is being linked up to api
- setup eslint to extend airbnb preset

### Testing Requirements
- [ ] Native app
  - check app builds locally
  - view time page
  - view settings page
